src/Engine/Manager/PhysicsManager.cpp

```diff
@@ -214,6 +214,10 @@ Utility::LockBox<Physics::Trigger> PhysicsManager::CreateTrigger(Component::Rigi
 
 void PhysicsManager::SetShape(Component::Shape* comp, std::shared_ptr<::Physics::Shape> shape) {
```

Comment:
Just a quick question here. Since we are not using smart pointers consistently throughout the project, is it really a good idea to use them at all? Concerning the risk of converting a smartpointer to a regular pointer.

Reply:
It would be good practice to use them everywhere. This one in particular uses reference counting to prevent memory from being freed when others still make use of it. As long as we don't call the `get` method haphazardly there should be a problem converting a smartpointer to a regular one.